### ISSUE TITLE
conflicting files : 'gitrob' and 'yawast'

### ISSUE THREAD

**Issue by philosophos** [BlackArch/blackarch#1580]
I know '--force' option , Just remind you.
`pacman -S blackarch`will show
```
......
error: failed to commit transaction (conflicting files)
/opt/ruby2.3/lib/ruby/gems/2.3.0/cache/colorize-0.8.1.gem exists in both 'gitrob' and 'yawast'
/opt/ruby2.3/lib/ruby/gems/2.3.0/cache/highline-1.7.8.gem exists in both 'gitrob' and 'yawast'
/opt/ruby2.3/lib/ruby/gems/2.3.0/gems/colorize-0.8.1/somefile exists in both 'gitrob' and 'yawast'
/opt/ruby2.3/lib/ruby/gems/2.3.0/gems/highline-1.7.8/somefile exists in both 'gitrob' and 'yawast'
/opt/ruby2.3/lib/ruby/gems/2.3.0/specifications/colorize-0.8.1.gemspec exists in both 'gitrob' and 'yawast'
/opt/ruby2.3/lib/ruby/gems/2.3.0/specifications/highline-1.7.8.gemspec exists in both 'gitrob' and 'yawast'
Errors occurred, no packages were upgraded.
```

**Comment by NovaCygni** [comment]
Again, pointless bug report, Anyone who knows how to use Linux, would know this "Error" is useless: Altering either package is a non-entity as they both require what they require, removing the Ruby Gems from one would mean that it cant work anymore, and not everyone installs "All" the tools....

**Comment by walle303** [comment]
The solution is to remove the ruby gems from both packages and package them separately as packages depended on by the packages.
Reporting this isn't pointless, but it has been reported already, albeit in a slightly different form [#1225], the fixes are being tracked by #1231
